features/dublin_core_settings.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Dublin Core settings
+  The CCA DIP access application will include a setting page, initially with
+  a couple of settings to configure the Dublin Core metadata fields used for
+  Collections and Folders.
+
+  Scenario: Setting page
+    Given an instance running
+    When an application user logs in the application
+    Then the user menu drop-down will show a link to go to a settings page
+      But only if the user is an Administrator # should we allow Editors and/or Managers too?
```

Comment:
I'm assuming this is a question for CCArchtects folk but I would say yes since these are the folks who will be doing the data entry work

Reply:
I'm not sure about managers, since they can't manage Collections or Folders at this point.